src/pip/_vendor/README.rst

```diff
@@ -8,6 +8,9 @@ Vendoring Policy
 * Vendored libraries **MUST** be released copies of libraries available on
   PyPI.
 
+* Libraries to be vendored **MUST** be available under a license that allows
```

Comment:
Maybe I'd stick with the previous wordings.
```suggestion
* Vendored libraries **MUST** be available under a license that allows
```

Reply:
Thank you for the review.  IIUC the other policies are referring to the downstream packages in `pip._vendor`, while the requirement I want to add here wants to specify the upstream libraries.

Reply:
I think @xavfernandez's suggestion is a good one -- the language consistency is good to have, and the difference between the two forms doesn't seem significant to me.